src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -55,9 +55,12 @@
 
 TEST_RESOURCE_PURGE_TIMEOUT = timedelta(hours=1)
 TEST_NIGHTLY_CI_RESOURCES_PURGE_TIMEOUT = timedelta(hours=3)  # Buffer for debugging nightly integration test runs
+# See https://docs.databricks.com/en/jobs/parameter-value-references.html#supported-value-references
 EXTRA_TASK_PARAMS = {
     "job_id": "{{job_id}}",
+    "job_name": "{{job.name}}",
     "run_id": "{{run_id}}",
+    "start_time": "{{job.start_time.iso_datetime}}",
```

Comment:
is it different from the info gotten from API?

Reply:
Combining my answer with this comment and the one above. See [this comment](https://github.com/databrickslabs/ucx/pull/2754#discussion_r1777319587) also.

I do not know if the time is different. However, what I like, is that this task avoids any API calls just like the `LogRecorder`. This would make it easier to add to any workflow. However, if we do not want that anyhow, we can also call the jobs API